index.js

```diff
@@ -24,6 +24,8 @@ hexo.config.marked = Object.assign({
   }
 }, hexo.config.marked);
 
+renderer.disableNunjucks = Boolean(hexo.config.marked.disableNunjucks);
```

Comment:
tested working in a dummy blog. somehow `marked.disableNunjucks = this.config.marked.disableNunjucks` didn't work for me, so I move it here instead.